src/common/dump.cpp

```diff
@@ -44,6 +44,39 @@ void dump_to_xflash(void) {
     w25x_fetch_error();
 }
 
+void dump_err_to_xflash(const char *error, const char *title) {
+    dumpmessage_t dump_message;
+    w25x_sector_erase(w25x_error_start_adress);
+    dump_message.not_displayed = 0; // validate message
+    strlcpy(dump_message.msg_title, title, sizeof(dump_message.msg_title));
+    strlcpy(dump_message.msg_content, error, sizeof(dump_message.msg_content));
+    w25x_program(w25x_error_start_adress, (uint8_t *)(&dump_message), sizeof(dumpmessage_t));
+    w25x_fetch_error();
+}
+
+int dump_in_xflash_get_message(char *msg_dst, uint16_t msg_dst_size, char *tit_dst, uint16_t tit_dst_size) {
```

Comment:
It looks like parameters `msg_dst_size` and `tit_dst_size` are unused. It is big mystery for me why are you not getting compiler warning for that.

Reply:
Oops, forgot that one. fixed in `e611ffc`